enhancements/oc/must-gather.md

```diff
@@ -101,12 +109,12 @@ If the `oc adm must-gather` tool's pod cannot be scheduled or run on the cluster
 ### Implementation Details/Notes/Constraints
 
 What are the caveats to the implementation? What are some important details that
-didn't come across above. Go in to as much detail as necessary here. This might
+didn't come across above? Go in to as much detail as necessary here. This might
 be a good place to talk about core concepts and how they releate.
 
 ### Risks and Mitigations
 
-What are the risks of this proposal and how do we mitigate. Think broadly. For
+What are the risks of this proposal and how do we mitigate? Think broadly. For
```

Comment:
In `Risks and Mitigations` section I'm expecting also to address problems with `--all-images`, such as:
1. not allowing to run more than 3 or 4 images at once
2. not allowing passing any arguments with it.